app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -104,7 +104,9 @@ def link_to_field(name, value, label = nil, facet_hash = {})
     # @see #index_field_link params
     # @return [Date]
     def human_readable_date(options)
-      Date.parse(options[:value]).to_formatted_s(:standard)
```

Comment:
Are you finding that the value of `options[:value]` is sometimes an `Array` and sometimes something else? If so, what's the something else? I'm wondering what the code paths into `human_readable_date` are, and whether we should do work upstream to ensure that `options[:value]` is consistently typed.